deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -0,0 +1,447 @@
+        </span>"""
+
+        displays = [headnote] + displays
+
+        return CheckResult(value=values_dict, display=displays, header='Train Test Label Drift')
+
+    def add_condition_drift_score_not_greater_than(self, max_allowed_psi_score: float = 0.2,
+                                                   max_allowed_earth_movers_score: float = 0.1):
+        """
+        Add condition - require drift score to not be more than a certain threshold.
+
+        The industry standard for PSI limit is above 0.2.
+        Earth movers does not have a common industry standard.
+
+        Args:
+            max_allowed_psi_score: the max threshold for the PSI score
+            max_allowed_earth_movers_score: the max threshold for the Earth Mover's Distance score
+
+        Returns:
+            ConditionResult: False if any column has passed the max threshold, True otherwise
+        """
+
+        def condition(result: Dict) -> ConditionResult:
+            drift_score = result['Drift score']
+            method = result['Method']
+            has_failed = (drift_score > max_allowed_psi_score and method == 'PSI') or \
+                         (drift_score > max_allowed_earth_movers_score and method == "Earth Mover's Distance")
+
+            if method == 'PSI' and has_failed:
+                return_str = f'Found label PSI above threshold: {drift_score:.2f}'
+                return ConditionResult(False, return_str)
+            elif method == "Earth Mover's Distance" and has_failed:
+                return_str = f'Label\'s Earth Mover\'s Distance above threshold: {drift_score:.2f}'
+                return ConditionResult(False, return_str)
+
+            return ConditionResult(True)
+
+        return self.add_condition(f'PSI <= {max_allowed_psi_score} and Earth Mover\'s Distance <= '
+                                  f'{max_allowed_earth_movers_score} for label drift',
+                                  condition)
+
+
+def get_bbox_area(label):
+    areas = (label.reshape((-1, 5))[:, 4] * label.reshape((-1, 5))[:, 3]).reshape(-1, 1).tolist()
+    return areas
+
+
+def count_num_bboxes(label):
+    num_bboxes = label.shape[0]
+    return num_bboxes
+
+
+def count_custom_transform_on_label(dataset: VisionDataset, transform: Callable = lambda x: x):
+    counter = Counter()
+    for i in range(len(dataset._data)):
+        list_of_arrays = next(iter(dataset._data))[1]
+        calc_res = [transform(arr) for arr in list_of_arrays]
+        if len(calc_res) != 0 and isinstance(calc_res[0], list):
+            calc_res = [x[0] for x in sum(calc_res, [])]
+        counter.update(calc_res)
+    return counter
+
+
+def histogram_in_batch(dataset: VisionDataset, transform: Callable = lambda x: x):
+    label_min = np.inf
+    label_max = -np.inf
+    for i in range(len(dataset._data)):
+        list_of_arrays = next(iter(dataset._data))[1]
+        calc_res = [transform(arr) for arr in list_of_arrays]
```

Comment:
feel like we should have a built in defense against failing transforms. Such as a try catch that makes sure we get null if fails so we can remove it and go on rather than crash.